hazelcast/src/main/java/com/hazelcast/config/alto/AltoConfig.java

```diff
@@ -1,35 +1,64 @@
 package com.hazelcast.config.alto;
 
-import com.hazelcast.config.InvalidConfigurationException;
+import com.hazelcast.spi.annotation.Beta;
 
 import java.util.Objects;
 
-public class AltoConfig {
-    private static final boolean DEFAULT_ENABLED = false;
-    private static final int DEFAULT_EVENTLOOP_COUNT = Runtime.getRuntime().availableProcessors();
+import static com.hazelcast.internal.util.Preconditions.checkPositive;
 
-    private boolean enabled = DEFAULT_ENABLED;
-    private int eventloopCount = DEFAULT_EVENTLOOP_COUNT;
+/**
+ * Hazelcast Alto is the next generation of Hazelcast built with thread
+ * per core architecture. It's still being developed and everything is
+ * subject to change. Alto is disabled by default.
+ *
+ * @since 5.3
+ */
+@Beta
+public class AltoConfig {
+    private boolean enabled = false;
+    private int eventloopCount = Runtime.getRuntime().availableProcessors();
 
+    /**
+     * @return true if alto is enabled
+     */
     public boolean isEnabled() {
         return enabled;
     }
 
+    /**
+     * @param enabled a boolean to enable or disable alto
+     * @return this alto configuration
+     */
     public AltoConfig setEnabled(boolean enabled) {
         this.enabled = enabled;
         return this;
     }
 
+    /**
+     * In Alto, everything is done in threads called Eventloop. This method
+     * gets the number of such threads. By default, it's equal to the
+     * number of available processors.
+     *
+     * @return the number of eventloop threads
+     * @see Runtime#availableProcessors()
+     * @see com.hazelcast.internal.tpc.Eventloop
+     */
     public int getEventloopCount() {
         return eventloopCount;
     }
 
+    /**
+     * In Alto, everything is done in threads called Eventloop. This method
```

Comment:
In the past the Eventloop was a thread, but the thread is now external from the eventloop.

Reply:
And if you have repeating explanations with getters and setters, I just let the getter refer to the setter to prevent duplication.